.github/workflows/release-docker-images.yaml

```
@@ -38,7 +38,7 @@ jobs:
         id: get-versions
         run: |
           if [[ ${{ github.event_name }} == 'schedule' ]]; then
-            version_list="\"v6.0.0\",\"v6.1.0\""
+            version_list="\"v6.0.0\",\"v6.0.1\",\"v6.1.0\""v6.1.1\""
```

```suggestion
            version_list="\"v6.0.1\",\"v6.1.1\""
```

You only need to include that latest bugfix release here. It would also be good to update the description and default version on lines 12-13.